astroquery/heasarc/core.py

```diff
@@ -286,7 +288,85 @@ def query_tap(self, query, *, maxrec=None):
         """
         log.debug(f'TAP query: {query}')
         self._saved_query = query
-        return self.tap.search(query, language='ADQL', maxrec=maxrec)
+        return self.tap.search(
+            query, language='ADQL', maxrec=maxrec, uploads=uploads)
```

Comment:
nitpick as it doesn't matter really, but just a FYI: we allow long (120) lines here, no need to do these kind of line breaks